roles/cloudman-boot/templates/cm_chart_values.yml.j2

```diff
@@ -19,26 +19,26 @@ cloudlaunch:
       postgresql:
          postgresqlPassword: {{ random_alphanumeric(16) }}
          postgresqlPostgresPassword: {{ random_alphanumeric(16) }}
-kubePrometheusStack:
-  influxdb:
-      setDefaultUser:
-          password: {{ random_alphanumeric(16) }}
-  prometheus:
-      alertmanager:
-          config:
-              global:
-                  http_config:
-                      basic_auth:
-                          password: {{ random_alphanumeric(16) }}
-      grafana:
-          adminPassword: {{ random_alphanumeric(16) }}
-          grafana.ini:
-             server:
-                root_url: "https://{{ cluster_hostname }}/grafana"
-             auth.generic_oauth:
-                auth_url: "https://{{ cluster_hostname }}/auth/realms/master/protocol/openid-connect/auth"
-                token_url: "https://{{ cluster_hostname }}/auth/realms/master/protocol/openid-connect/token"
-                api_url: "https://{{ cluster_hostname }}/auth/realms/master/protocol/openid-connect/userinfo"
+influxdb:
+    setDefaultUser:
+        user:
+            password: {{ random_alphanumeric(16) }}
+kube-prometheus-stack:
```

Comment:
This should be kubePrometheusStack I believe, or the dash trips things up?

Reply:
I was trying with the version that is currently on master following: https://github.com/CloudVE/cloudman-helm/pull/75
The camelcase is not ideal cause it then gets used for the release name and it's not DNS compliant. I've defaulted to `kubeprometheus` as an alias instead for now.